app/src/main/java/com/guzelgimadieva/tasky/authorization/ui/LoginEvent.kt

```diff
@@ -0,0 +1,18 @@
+package com.guzelgimadieva.tasky.authorization.ui
+
+sealed interface LoginEvent {
+
+    data class EmailChanged(
+        val email: String
+    ) : LoginEvent
+
+    data class PasswordChanged(
+       val password: String
+    ) : LoginEvent
+
+    data class PasswordVisibilityChanged(
+        val isPasswordVisible: Boolean
+    ) : LoginEvent
+
+    data class Login(val email: String, val password: String) : LoginEvent
```

Comment:
When sending the Login event to the ViewModel, the VM already knows the value of the current email and password state as it hosts your complete screen state. This can be a `data object` therefore which just triggers the login function in the VM